docs/best-practices/data-import.rst

```diff
@@ -371,11 +386,12 @@ adapt to the increasing data volume! Simply use ``ALTER TABLE users SET
   - Table creation of `PARTITIONED BY clause`_
   - `Alter a partitioned table`_
 
+
 Summary
 =======
 
 To sum up the points described above, importing huge datasets is not difficult
-if a few things are kept in mind. These are:
+as long as a few things are kept in mind. These are:
```

Comment:
perhaps "providing" is easier for non-native speakers?

Reply:
I'd think that's less easy! Compare Dutch: "zolang", German: "solange", Spanish: "siempre y cuando", etc...

Reply:
okay!